src/components/spreadsheet-view/spreadsheet-tabs/toolbar/PartialLoadingMenuButton.tsx

```diff
@@ -116,21 +118,21 @@ export default function PartialLoadingMenuButton({ disabled, ...props }: Readonl
 
     return (
         <>
-            <TooltipIconButton
-                id="lazy-loading-button"
-                aria-controls={open ? 'lazy-loading-menu' : undefined}
-                aria-haspopup="true"
-                aria-expanded={open ? 'true' : undefined}
-                tooltip={<FormattedMessage id="spreadsheet/tabs/lazy_loading/toolbar_button_tooltip" />}
-                size="small"
-                disabled={disabled}
-                onClick={handleClick}
-                {...props}
-            >
-                <Badge variant="dot" color="secondary" invisible={!isOptionalData}>
-                    <BtnIcon color={open ? 'action' : 'primary'} />
-                </Badge>
-            </TooltipIconButton>
+            <Badge variant="dot" color="secondary" invisible={!isOptionalData}>
+                <Tooltip title={<FormattedMessage id="spreadsheet/tabs/lazy_loading/toolbar_button_tooltip" />}>
+                    <span>
+                        <Button
+                            sx={spreadsheetStyles.toolbarButton}
+                            size={'small'}
+                            onClick={handleClick}
+                            disabled={open || disabled}
```

Comment:
Do we really want to disable it when opened ? It was its behaviour so far but it's the only toolbar button to behave that way I think